benefits/eligibility/templates/eligibility/start.html

```diff
@@ -6,11 +6,9 @@
 
 {% block main_content %}
   <div class="container">
-    <h1 class="headline">{% translate "eligibility.pages.start.content_title" %}</h1>
-    <p>{% blocktranslate with info_link=info_link%}eligibility.pages.start.p[0]{{ info_link }}{% endblocktranslate %}</p>
-    <p>{% translate "eligibility.pages.start.p[1]" %}</p>
+    <h1 class="headline">{{ start_title }}</h1>
 
-    <h2 class="media-title">{{ title }}</h2>
+    <h2 class="media-title">{{ start_content_title }}</h2>
```

Comment:
Another naming option here: We could start using `h1`, `h2`??? @thekaveman

Reply:
Or maybe that's too tied to the code. Headline, sub-headline is another option to denote that one is lower in hierarchy.

Reply:
Yeah true. Either way works, so long as we are consistent!

Reply:
I was also bothered by the inconsistency. I appreciate the feedback and suggestions